### ISSUE TITLE
Nightly builds don't bundle frontend.

### ISSUE THREAD

**Issue by wwitzel3** [vmware-tanzu/octant#368]
The nightly builds are missing the dist/dash-frontend box.

The frontend build appears to be running.

![image (9)](https://user-images.githubusercontent.com/27317/67522502-de728180-f67a-11e9-8d00-12bf3f43d613.png)

```
2019-10-24T16:13:48.566-0400    INFO    dash/dash.go:366        Using embedded Octant frontend                                                                                                                               
2019-10-24T16:13:48.566-0400    DEBUG   dash/dash.go:191        running dashboard service: could not locate box "dist/dash-frontend"  
```

Though when testing the `.goreleaser-nightly.yml` locally, it appears to work just fine.

```
/home/wwitzel3/repos/octant/dist/octant_2019-10-24T20:12:12Z_Linux-64bit> ./octant -v
...
2019-10-24T16:15:36.236-0400    INFO    dash/dash.go:366        Using embedded Octant frontend                                                                                                                               
2019-10-24T16:15:36.236-0400    INFO    dash/dash.go:345        Dashboard is available at http://127.0.0.1:7777
```

**Comment by GuessWhoSamFoo** [comment]
https://discourse.drone.io/t/share-workspace-between-two-different-drone-pipelines/2143

Since the addition of nightly, the release step was moved into a separate pipeline which does not share a workspace by default. We will need to keep the output of `make frontend` in a shared temporary volume between pipelines.

**Comment by GuessWhoSamFoo** [comment]
Closed via #371
